include/sp_vm_api.h

```diff
@@ -982,6 +995,20 @@ namespace SourcePawn
      * @return          Plugin context.
      */
     virtual IPluginContext *Context() const = 0;
+
+    /**
+     * @brief Return the error code of the error report.
+     *
+     * @return          Integer code.
+     */
+    virtual int Code() const = 0;

```

Comment:
If possible, I would drop this from the public API. SM isn't supposed to know about codes anymore, since they're kind of dumb.